server/issue_parser.go

```diff
@@ -20,6 +20,7 @@ func parseJiraLinksToMarkdown(text string) string {
 }
 
 func mdKeySummaryLink(issue *jira.Issue) string {
+	// Use Self URL only to extract the full hostname from it
```

Comment:
What has changed about the Self URL's?

Reply:
They no longer include /issue/ in the path, don't ask me why...